src/createServer.js

```diff
@@ -1,3 +1,73 @@
+/* eslint-disable no-useless-escape */
+/* eslint-disable quotes */
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const PORT = process.argv.PORT || 3006;
+
+const caseName = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const errorMessage = {
+  missingText: `Text to convert is required. Correct request is: \"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>\".`,
+  missingCase: `"toCase" query param is required. Correct request is: \"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
+  invalidCase: `This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.`,
+};
+
+const createServer = () => (
+  http.createServer((req, res) => {
+    res.writeHead(200, { 'Content-Type': 'application/json' });
```

Comment:
How do you know for sure that it's going to succeed? I think it's better to assign status code right before sending the response